entry/src/main/ets/MainAbility/pages/indexx.ets

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import prompt from '@system.prompt';
+import { Tab, TabBarOptions, TabBar, TabBarType, IconType, TabBarModel }  from '@ohos/material-tabs'
+
+@Entry
+@Component
+struct Index {
+  private options: TabBarOptions [] = [
+    new TabBarOptions(1, $r('app.string.tab_1'), $r('app.media.icon')),
```

Comment:
Test the scenario of adding same ids to the different TabBarOptions and ensure there is no crash or any major abnormal behavior